03_simon_says/simon_says.rb

```diff
@@ -1 +1,34 @@
-#write your code here
+def echo(str)
+    str
+end
+
+def shout(str)
+    str.upcase
+end
+
+def repeat(str, n=2)
+    ("#{str} " * n).strip
```

Comment:
Fortunately, none of the test cases involve passing in strings with leading or trailing whitespace, but if they did, #strip would remove more characters than the one trailing character in `"#{str} "`. One way to fix this:

```suggestion
    ("#{str} " * n).delete_suffix(" ")
```